ros/src/twist_controller/dbw_node.py

```diff
@@ -39,6 +40,9 @@ def __init__(self):
         max_lat_accel = rospy.get_param('~max_lat_accel', 3.)
         max_steer_angle = rospy.get_param('~max_steer_angle', 8.)
 
+        # TODO - define min_speed as a parameter in the parameter server?
+        min_speed = 0.
```

Comment:
We're starting to try using the param server in the Waypoint Updater code but it's a little tricky.  Check out #13.